### PR TITLE
added some initial logic to load the teacher logits

### PR DESCRIPTION
**Pull request by shamanez**
- Initial logic to load teacher logits
- At the moment, this is only for the logit-based training.

### CODE REVIEW

**Comment by mertege** [comment]
Hi @shamanez, I created offline logits, and I saved them as "logits". I then loaded this dataset and checked that I could see the "logits" key in the relevant dataset before SFTTrainer. However, in the "compute_loss" function, "inputs" only contains the "input_ids" and "attention_mask" keys. Since there are none of the "logits" in "compute_loss", I cannot get the teacher logits. I think that trl SFTTrainer only gets inputs with "input_ids" and "attantion_mask" keys. Have you encountered this kind of problem?

I appreciate any help you can provide.

**Comment by shamanez** [comment]
can you try    `remove_unused_columns=False` in the SFT trainer.

**Comment by shing100** [comment]
Hello, I tried to make teacher data through the code you posted. I tried to make it using the llama3.1-70B-Inst model, but only RAM memory goes up and GPUs(80GB*8) doesn't work and I get an error.

I ask for your help.

used commend : accelerate launch teacher_first_logits.py
